install/install.go

```diff
@@ -1051,6 +1405,68 @@ func K8sUninstaller(c *k8s.Client, o Options) error {
 	return nil
 }
 
+// K8sUninstaller for karmor uninstall
+func K8sUninstaller(c *k8s.Client, o Options) error {
+	var ns string
+
+	if o.Namespace != "" {
+		pods, _ := c.K8sClientset.CoreV1().Pods(o.Namespace).List(context.TODO(), metav1.ListOptions{LabelSelector: "kubearmor-app", FieldSelector: "status.phase==Running"})
+		if len(pods.Items) == 0 {
+			fmt.Println("❌\tKubeArmor is either not installed or you have specified incorrect namespace.\nℹ️\tPlease ensure you have installed KubeArmor or have specificed the correct namespace.")
+			return nil
+		}
+		ns = o.Namespace
+	} else {
+		if pods, _ := c.K8sClientset.CoreV1().Pods("").List(context.TODO(), metav1.ListOptions{LabelSelector: "kubearmor-app", FieldSelector: "status.phase==Running"}); len(pods.Items) == 0 {
+			fmt.Println("❌\tKubeArmor is not installed.\nℹ️\tPlease ensure you have installed KubeArmor.")
+			return nil
+		} else {
+			if len(pods.Items) > 0 {
+				ns = pods.Items[0].Namespace
+			}
+		}
+	}
+
+	settings := cli.New()
+	settings.SetNamespace(ns)
+	settings.RESTClientGetter()
+
+	actionConfig := new(action.Configuration)
+	if err := actionConfig.Init(settings.RESTClientGetter(), ns, os.Getenv("HELM_DRIVER"), log.Printf); err != nil {
+		log.Printf("%+v", err)
+		os.Exit(1)
+	}
+
+	client := action.NewUninstall(actionConfig)
+	client.Timeout = 5 * time.Minute
+	client.DeletionPropagation = "background"
+
+	log.SetOutput(io.Discard)
+	_, err := client.Run("kubearmor-operator")
+	log.SetOutput(os.Stdout)
+
+	if err != nil {
+		fmt.Println("ℹ️  Helm release not found. Switching to legacy uninstaller.")
+		return err
+	}
+
+	operatorClientSet, err := operatorClient.NewForConfig(c.Config)
+	if err != nil {
+		return fmt.Errorf("failed to create operator client: %w", err)
+	}
+	if err := operatorClientSet.OperatorV1().KubeArmorConfigs(ns).Delete(context.Background(), "kubearmorconfig-default", metav1.DeleteOptions{}); err != nil {
```

Comment:
We might not want to return an error if the resource itself was not found. Let's use [IsNotFound](https://pkg.go.dev/k8s.io/apimachinery/pkg/api/errors#IsNotFound) so that rest of the process can continue.